admin_guide/diagnostics_tool.adoc

```diff
@@ -116,37 +116,60 @@ current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
 
-[[additional-cluster-health-checks]]
-== Additional Diagnostic Checks via Ansible
-
-// TODO: add link to OCP image once it is available
-
-Some additional diagnostic checks are available through the *openshift-ansible*
-container image. See the image's link:https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[source repository] for usage information.
-
-The following health checks belong to a diagnostic task meant to be run against
-the Ansible inventory file for a deployed {product-title} cluster. They can
-report common problems for the current {product-title} installation.
+[[ansible-based-tooling-health-checks]]
+== Ansible-based Health Checks
+
+// tag::ansible-based-health-checks-intro[]
+Additional diagnostic health checks are available through the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Ansible-based tooling] used to install and manage {product-title} clusters. They can report
+common deployment problems for the current {product-title} installation.
+
+These checks can be run either using the `ansible-playbook` command (the same
+method used during
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation]) or using the Docker CLI to directly run a
+link:https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[containerized version] of *openshift-ansible*. For the `ansible-playbook` method, the checks
+are provided by the
+ifdef::openshift-enterprise[]
+*atomic-openshift-utils* RPM package.
+endif::[]
+ifdef::openshift-origin[]
+xref:../install_config/install/host_preparation.adoc#preparing-for-advanced-installations-origin[*openshift-ansible*]
+Git repository.
+endif::[]
+For the Docker CLI method,
+ifdef::openshift-enterprise[]
+the *openshift3/ose-ansible* container image is distributed via the
+link:https://registry.access.redhat.com[Red Hat Container Registry].
+endif::[]
+ifdef::openshift-origin[]
+the *openshift/origin-ansible* container image is distributed via Docker Hub.
+endif::[]
+// end::ansible-based-health-checks-intro[]
+Example usage for each method are provided in subsequent sections.
+
+The following health checks are a set of diagnostic tasks that are meant to be
+run against the Ansible inventory file for a deployed {product-title} cluster
+using the provided *_health.yml_* playbook.
+
+[WARNING]
+====
+Due to potential changes the health checks could make to hosts, they should only
+be used on clusters that have been deployed using Ansible and using the same
+inventory file with which it was deployed. Changes mostly involve installing
```

Comment:
Err... run the playbook and see if it changes anything?

Seeing what would be changed without making the changes is what the `--check` ansible flag is for but I honestly don't know how accurate or useful it is with as much custom stuff as openshift-ansible includes. Could be fine, could be broken, could even make changes anyway. I don't think it's a use case anyone's paying attention to.